qtbot with PySide2 fails in github actions

hi,
Hi the following bug in github actions  CI not locally:
```
2020-04-08T10:54:24.2278599Z ##[group]Run python -m pytest -vvv -s
2020-04-08T10:54:24.2279416Z [36;1mpython -m pytest -vvv -s[0m
2020-04-08T10:54:24.2321201Z shell: /bin/bash -e {0}
2020-04-08T10:54:24.2321307Z env:
2020-04-08T10:54:24.2321402Z   DISPLAY: :99.0
2020-04-08T10:54:24.2321482Z   pythonLocation: /opt/hostedtoolcache/Python/3.7.6/x64
2020-04-08T10:54:24.2321584Z ##[endgroup]
2020-04-08T10:54:24.6769440Z ============================= test session starts ==============================
2020-04-08T10:54:24.6770421Z platform linux -- Python 3.7.6, pytest-5.3.2, py-1.8.1, pluggy-0.13.1 -- /opt/hostedtoolcache/Python/3.7.6/x64/bin/python
2020-04-08T10:54:24.6778989Z cachedir: .pytest_cache
2020-04-08T10:54:24.6779452Z PySide2 5.14.1 -- Qt runtime 5.14.1 -- Qt compiled 5.14.1
2020-04-08T10:54:24.6779578Z rootdir: /home/runner/work/MyCartable/MyCartable
2020-04-08T10:54:24.6779806Z plugins: qt-3.3.0
2020-04-08T10:54:29.2367912Z collecting ... collected 1 item
2020-04-08T10:54:29.2371413Z 
2020-04-08T10:54:29.5635621Z Fatal Python error: Aborted
2020-04-08T10:54:29.5636386Z 
2020-04-08T10:54:29.5636717Z Current thread 0x00007f29af350680 (most recent call first):
2020-04-08T10:54:29.5637935Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pytestqt/plugin.py", line 57 in qapp
2020-04-08T10:54:29.5638869Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 787 in call_fixture_func
2020-04-08T10:54:29.5639674Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 957 in pytest_fixture_setup
2020-04-08T10:54:29.5640725Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
2020-04-08T10:54:29.5641520Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
2020-04-08T10:54:29.5642447Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
2020-04-08T10:54:29.5643218Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
2020-04-08T10:54:29.5644061Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 908 in execute
2020-04-08T10:54:29.5644857Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 579 in _compute_fixture_value
2020-04-08T10:54:29.5646315Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 498 in _get_active_fixturedef
2020-04-08T10:54:29.5647420Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 482 in getfixturevalue
2020-04-08T10:54:29.5648200Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 472 in _fillfixtures
2020-04-08T10:54:29.5648954Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/fixtures.py", line 293 in fillfixtures
2020-04-08T10:54:29.5649714Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/python.py", line 1439 in setup
2020-04-08T10:54:29.5650562Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 364 in prepare
2020-04-08T10:54:29.5651703Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 119 in pytest_runtest_setup
2020-04-08T10:54:29.5652460Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
2020-04-08T10:54:29.5653656Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
2020-04-08T10:54:29.5654687Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
2020-04-08T10:54:29.5655769Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
2020-04-08T10:54:29.5658033Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 207 in <lambda>
2020-04-08T10:54:29.5658900Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 234 in from_call
2020-04-08T10:54:29.5659717Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 207 in call_runtest_hook
2020-04-08T10:54:29.5660392Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 182 in call_and_report
2020-04-08T10:54:29.5661048Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 90 in runtestprotocol
2020-04-08T10:54:29.5661714Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/runner.py", line 81 in pytest_runtest_protocol
2020-04-08T10:54:29.5662367Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
2020-04-08T10:54:29.5663014Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
2020-04-08T10:54:29.5663710Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
2020-04-08T10:54:29.5664356Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
2020-04-08T10:54:29.5665159Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/main.py", line 270 in pytest_runtestloop
2020-04-08T10:54:29.5665796Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
2020-04-08T10:54:29.5666588Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
2020-04-08T10:54:29.5673468Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
2020-04-08T10:54:29.5674585Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
2020-04-08T10:54:29.5675579Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/main.py", line 246 in _main
2020-04-08T10:54:29.5676250Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/main.py", line 196 in wrap_session
2020-04-08T10:54:29.5676742Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/main.py", line 239 in pytest_cmdline_main
2020-04-08T10:54:29.5677212Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/callers.py", line 187 in _multicall
2020-04-08T10:54:29.5677669Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 87 in <lambda>
2020-04-08T10:54:29.5678125Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/manager.py", line 93 in _hookexec
2020-04-08T10:54:29.5678599Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pluggy/hooks.py", line 286 in __call__
2020-04-08T10:54:29.5679069Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/_pytest/config/__init__.py", line 92 in main
2020-04-08T10:54:29.5679525Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/site-packages/pytest/__main__.py", line 7 in <module>
2020-04-08T10:54:29.5679746Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/runpy.py", line 85 in _run_code
2020-04-08T10:54:29.5679971Z   File "/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/runpy.py", line 193 in _run_module_as_main
2020-04-08T10:54:39.0807343Z /home/runner/work/_temp/9af4350e-c569-4f21-afd2-8566d9a0c474.sh: line 1:  2584 Aborted                 (core dumped) python -m pytest -vvv -s
2020-04-08T10:54:39.0807950Z tests/test_main.py::test_un 
2020-04-08T10:54:39.0813192Z ##[error]Process completed with exit code 134.
2020-04-08T10:54:39.0827961Z Cleaning up orphan processes
2020-04-08T10:54:39.1122098Z Terminate orphan process: pid (2582) (Xvfb)
```

the only test is  (no import in file):
```
def test_un(qtbot):
    assert True

```
conftest file empty.

Not sure if it could be related to #202 or #206

thanks for help

--- a/docs/troubleshooting.rst
+++ b/docs/troubleshooting.rst
@@ -60,3 +60,37 @@ If you are running your code on Travis-CI make sure that your ``.travis.yml`` ha
 More details can be found in `issue #206`_.
 
 .. _issue #206: https://github.com/pytest-dev/pytest-qt/issues/206
+
+GitHub Actions
+----------------
+
+When using ``ubuntu-latest`` on Github Actions, the package ``libxkbcommon-x11-0`` has to be installed, ``DISPLAY`` should be set and ``xvfb`` run. More details can be found in `issue #293`_.
+
+.. _issue #293: https://github.com/pytest-dev/pytest-qt/issues/293
+
+As an example, here is working config :
+
+.. code-block:: yaml
+
+    name: my qt ci in github actions
+    on: [push, pull_request]
+    jobs:
+      Linux:
+        runs-on: ${{ matrix.os }}
+        strategy:
+          matrix:
+            os : [ubuntu-latest]
+            python: [3.7]
+        env:
+          DISPLAY: ':99.0'
+        steps:
+        - name: get repo
+          uses: actions/checkout@v1
+        - name: Set up Python
+          uses: actions/setup-python@v1
+          with:
+            python-version: ${{ matrix.python }}
+        - name: setup ${{ matrix.os }}
+          run: |
+            sudo apt install libxkbcommon-x11-0
+            /sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -screen 0 1920x1200x24 -ac +extension GLX

--- a/docs/troubleshooting.rst
+++ b/docs/troubleshooting.rst
@@ -68,7 +68,7 @@ When using ``ubuntu-latest`` on Github Actions, the package ``libxkbcommon-x11-0
 
 .. _issue #293: https://github.com/pytest-dev/pytest-qt/issues/293
 
-As an example, here is working config :
+As an example, here is a working config :
 
 .. code-block:: yaml
 

